src/SUMMARY.md

```diff
@@ -138,10 +138,11 @@
     - [Two-phase-borrows](./borrow_check/two_phase_borrows.md)
 - [Parameter Environments](./param_env.md)
 - [Errors and Lints](diagnostics.md)
-    - [Creating Errors With SessionDiagnostic](./diagnostics/sessiondiagnostic.md)
```

Comment:
Maybe we could keep that page and redirect to the new places so that we don't break someone's links.

Reply:
Fixed using `output.html.redirect` in `book.toml`.